npc_gzip/__init__.py

```diff
@@ -0,0 +1,4 @@
+from . import compressors
+from .distance import *
+from .exceptions import *
+from .utils import *
```

Comment:
Does the packaged version support a command-line syntax like the original codebase does, where a command/script is available that allows the software to be used not just as a library, but also to be effectively run as a program, perhaps in a way similar to how `main_text.py` is currently documented in the readme?

That seems like it may be important, and I didn't notice it during the course of reviewing. `git grep ArgumentParser` also only reveals `original_codebase/main_text.py`.

This could be either as a script or just with `python -m`, but I'd suggest having a script, which can actually be done even more nicely than before, using https://python-poetry.org/docs/pyproject/#scripts.

Reply:
It does not currently, but that should be simple to add. Do you think it's worth saving for `0.1.1`?

Reply:
I think either way is fine. But if it doesn't make the initial PyPI release then I think the readme should make sure to prominently show instructions for `original_codebase/main_text.py`, since many users will likely want to run it as a script.

If omitting it from the first release lets the first release come out significantly earlier, then I'd lean toward saying that's the slightly better way to go, based on the idea that it may be more manageable to have more frequent releases with smaller changes in each release.

Reply:
The original codebase should still work as written.

The `examples/` directory shows how to use the package as a python module so I think we're good for 0.1.0. 

0.1.1 can be a quick patch release that adds CLI support

Reply:
Sounds good. I'll mark this comment as resolved since it requires no changes in this PR.